coralproject/talk
# [CORL-2730] give buttons clearer labels for screen readers

## What does this PR do?
This PR adds more context to the "Change" buttons for email, username, and password.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags?
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. Enable your screenreader of choice
2. In a stream, navigate to My Profile > Account
3. Tab through this pane, and when focusing on the buttons for changing email, username, and password, observe that the screen reader reads "Change email", "Change username", and "Change password", respecitively.

## Where any tests migrated to React Testing Library?
No.

## How do we deploy this PR?
No special considerations needed.
